openstack/maia/templates/seed.yaml

```diff
@@ -0,0 +1,92 @@
+{{- if .Release.IsUpgrade }}
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: maia-seed
+  labels:
+    component: maia
+spec:
+  requires:
+#    - keystone-seed
+#    - monsoon3/domain-cc3test-seed
+#    - monsoon3/domain-ccadmin-seed
+
+  roles:
+  - monitoring_viewer
+  - monitoring_admin
+
+# TODO expose as OpenStack service
```

Comment:
yes, for consumption by Elektra. Not today, but tomorrow --> TODO

The roles are needed today, therefore they are not commented out